protocols/raft/src/main/java/io/atomix/protocols/raft/roles/FollowerRole.java

```diff
@@ -59,39 +55,38 @@ public FollowerRole(RaftContext context) {
 
   @Override
   public synchronized CompletableFuture<RaftRole> open() {
-    return super.open().thenRun(this::startHeartbeatTimeout).thenApply(v -> this);
+    raft.setLastHeartbeatTime();
+    return super.open().thenRun(this::startHeartbeatTimer).thenApply(v -> this);
   }
 
   /**
    * Starts the heartbeat timer.
    */
-  private void startHeartbeatTimeout() {
+  private void startHeartbeatTimer() {
     log.trace("Starting heartbeat timer");
+    AtomicLong lastHeartbeat = new AtomicLong();
```

Comment:
This value needs to be updated, otherwise the last heartbeat time is always reported.